source/features/rgh-feature-descriptions.tsx

```diff
@@ -22,7 +22,7 @@ async function add(infoBanner: HTMLElement): Promise<void> {
 	conversationsUrl.searchParams.set('q', `sort:updated-desc "${feature.id}"`);
 
 	infoBanner.before(
-		<div className="Box mb-3">
+		<div className="Box mb-3 d-inline-block width-full">
```

Comment:
Thank you! Can you also add a comment? It's not immediately clear why they're there
```suggestion
		{/* Block and width classes required to avoid margin collapse */}
		<div className="Box mb-3 d-inline-block width-full">
```